docs/includes/topic_f5lbaas-l7_content_switching.rst

```diff
@@ -0,0 +1,197 @@
+            +----------------------+-------------------------+------------------------------------------+
+            | CONTAINS             | No                      | Contains                                 |
+            +----------------------+-------------------------+------------------------------------------+
+            | CONTAINS             | Yes                     | Does not contain                         |
+            +----------------------+-------------------------+------------------------------------------+
+            | REGEX                | X                       | No direct mapping                        |
+            +----------------------+-------------------------+------------------------------------------+
+
+        All L7Policy actions map directly to TMOS traffic policy rule actions:
+
+            +------------------+---------------------------------+
+            | L7 Policy Action | TMOS Traffic Policy Rule Action |
+            +==================+=================================+
+            | Reject           | Reset traffic                   |
+            +------------------+---------------------------------+
+            | RedirectToUrl    | Redirect                        |
+            +------------------+---------------------------------+
+            | RedirectToPool   | Forward traffic to pool         |
+            +------------------+---------------------------------+
+
+Since the L7 Rules are expected to match on each HTTP request, if a listener has an L7Policy attached, the presence of a OneConnect profile on the corresponding TMOS virtual server must be assured.
+
+Initially, the support for Regex is not available.  Since there is a direct mapping of the other compare types in Neutron to BIG-IP LTM, and regular expressions can in some cases be expressed as the logical ‘OR’ of multiple string compares, we propose an initial implementation of L7 Policies and Rules without regular expressions with the understanding that it will be available as a feature in a future release.
+
+Prerequisites
+-------------
+
+- Licensed, operational BIG-IP :term:`device` or :term:`device cluster`.
+- Operational OpenStack cloud (|openstack| release).
+- Administrator access to both BIG-IP device(s) and OpenStack cloud.
+- F5 :ref:`agent <agent:home>` and :ref:`service provider driver <Install the F5 LBaaSv2 Driver>` installed on the Neutron controller and all other hosts from which you want to provision LBaaS services.
+- Knowledge of `OpenStack Networking <http://docs.openstack.org/mitaka/networking-guide/>`_ concepts.
+- Knowledge of BIG-IP `system configuration`_, `local traffic management`_, & `device service clustering`_.
+
+Caveats
+-------
+
+- As mentioned previously, the REGEX comparison type is not available in the current release.
+
+Configuration
+-------------
+
+#. No additional configuration needed in the :ref:`Agent Configuration File`.
```

Comment:
"It's not necessary to make any agent configuration changes. Rather, L7 switching policy and rule definitions are made when creating or updating a listener, as shown in the example below from the OpenStack documentation. "